doc/conf.py

```diff
@@ -270,7 +277,7 @@
     "footer_center": [],
     "footer_end": [],
     # Use :html_theme.sidebar_secondary.remove: for file-wide removal
-    "secondary_sidebar_items": ["page-toc", "sourcelink"],
+    "secondary_sidebar_items": {"**": ["page-toc", "sourcelink"]},
```

Comment:
Can you please add an inline comment to explain the meaning of the `**` key in this dict?